src/fragments/metadata.ts

```diff
@@ -0,0 +1,16 @@
+import gql from "graphql-tag";
+
+export const metadataFragment = gql`
+  fragment MetadataItem on MetadataItem {
+    key
+    value
+  }
+  fragment Metadata on ObjectWithMetadata {
```

Comment:
```suggestion
  fragment MetadataFragment on ObjectWithMetadata {
```

Reply:
I'll change it when all PRs would be merged, so I'll be doing it once